Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -120,22 +122,19 @@ public function interpret(array $data, $associative = true)
                 $entities[$index] = $entity;
                 $this->internalLookupCache[] = $entity;
             } catch (InterpreterException $exception) {
-                $this->handleInterpreterError($exception, $index, $row);
+                $exceptionEvent = new ExceptionEvent($this->log, $exception, $index);
```

Comment:
In this class, the `$log` variable is needed only to pass as an argument for event object constructor.
I think it might be the same case for other classes.
Maybe creating factory to create these events, with `$log` object inside, would be nicer way.